tests/test_validation.py

```diff
@@ -0,0 +1,38 @@
+from django.core.exceptions import ValidationError
+
+import pytest
+
+from zgw_consumers.models.validators import NonUrlValidator, PrefixValidator
+
+
+def test_start_with_validator_starts_with_false():
+    validator = PrefixValidator(prefix="/", starts_with=False)
+
+    assert validator.__call__("no_leading_slash") is None
```

Comment:
the return value of validators is not relevant, what you actually want to test is the exception behaviour:

```suggestion
    try:
        validator("no_leading_slash")
    except ValidationError:
        pytest.fail("Expected validation to pass")
```